src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AvoidMultipleEnumerations/AvoidMultipleEnumerations.WellKnownMethodInfo.cs

```diff
@@ -1,68 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
```

Comment:
This file is renamed and changed to non-nested class. No real change

Reply:
I don't know whether we care much about it but it might have been easier to do a `git move` command to help git understand the change.